### ISSUE TITLE
[IntelliJ][ReportedByUser] Uncaught Exception  com.microsoft.azure.toolkit.lib.common.operation.OperationException: load Resource group (rg-app-azure-function-examples-220603145106)

### ISSUE THREAD

**Issue by svaareddy** [microsoft/azure-tools-for-java#6672]
IntelliJ build version: 2022.1.2 IU-221.5787.30
OS: Mac OS X
JDK: JetBrains s.r.o. 11.0.15
Plugin version: 3.65.0-2022.1
Additional Info: None
Parent component: 
```
com.intellij.openapi.wm.impl.IdeRootPane[,0,0,1680x1050,layout=javax.swing.JRootPane$RootLayout,alignmentX=0.0,alignmentY=0.0,border=,flags=449,maximumSize=,minimumSize=,preferredSize=]
```
Error message: 
```
IdeaLoggingEvent[message=null, throwable=com.microsoft.azure.toolkit.lib.common.operation.OperationException: load Resource group (rg-app-azure-function-examples-220603145106)
	at com.microsoft.azure.toolkit.lib.common.operation.AzureOperationAspect.afterThrowing(AzureOperationAspect.java:88)
	at com.microsoft.azure.toolkit.lib.common.operation.AzureOperationAspect.afterThrowing(AzureOperationAspect.java:49)
	at com.microsoft.azure.toolkit.lib.common.model.AbstractAzResourceModule.loadResourceFromAzure(AbstractAzResourceModule.java:425)
	at com.microsoft.azure.toolkit.lib.common.model.AbstractAzResourceModule.get(AbstractAzResourceModule.java:196)
	at com.microsoft.azure.toolkit.lib.common.model.AbstractAzResourceModule.getOrDraft(AbstractAzResourceModule.java:250)
	at com.microsoft.azure.toolkit.lib.resource.ResourceGroupConfig.toResource(ResourceGroupConfig.java:43)
	at com.microsoft.azure.toolkit

**Comment by wangmingliang-ms** [comment]
this error should be fixed by commit [9220142](https://github.com/microsoft/azure-tools-for-java/commit/92201420ede004560832f902620c99ea177efb82), the fixed will included in the next stable release.

**Comment by wangmingliang-ms** [comment]
close as it has been fixed since v3.69.0, you can upgrade to the latest and try. Feel free to reopen it when you encounter this error again.
